tests/testthat/test-renderer1-tooltip.R

```diff
@@ -82,8 +82,9 @@ test_that("tooltip shows correct content for rect", {
   center_y <- rect_y + (rect_height / 2)
   
   # Simulate hover over rectangle center
-  remDr$Input$dispatchMouseEvent(type = "mouseMoved", x = center_x, y = center_y)
-  Sys.sleep(0.5) # Wait for tooltip
+  remDr$Input$dispatchMouseEvent(type = "mouseMoved", x = center_x + 10, y = center_y)
```

Comment:
please remove the +10 here, which is another magic number. (and below, and +5 etc)
instead can you please try using something like https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect ?
```r
runtime_evaluate(script=...getBoundingClientRect())
```
etc